src/orderbook.js

```diff
@@ -1185,6 +1309,14 @@ OrderBook.prototype.computeAutobridgedOffers = function() {
   assert(!this._currencyGets.is_native() && !this._currencyPays.is_native(),
     'Autobridging is only for IOU:IOU orderbooks');
 
+  if (this._destroyed) {
+    return;
+  }
+
+  if (!this._gotOffersFromLegOne || !this._gotOffersFromLegTwo) {
```

Comment:
`_gotOffersFromLegOne` and `_gotOffersFromLegTwo` will both be true when data from both legs will arrive